src/libs/ReportUtils.ts

```diff
@@ -4312,7 +4320,8 @@ function getNextApproverAccountID(report: OnyxEntry<Report>, isUnapproved = fals
 
     const nextApproverEmail = approvalChain.length === 1 ? approvalChain.at(0) : approvalChain.at(approvalChain.indexOf(currentUserEmail ?? '') + 1);
     if (!nextApproverEmail) {
-        return submitToAccountID;
+        // If there's no next approver in the chain, return undefined to indicate the current user is the final approver
+        return undefined;
```

Comment:
OK, this change caused a regression as expected even after testing I missed one. 

https://github.com/Expensify/App/issues/72742.  @fahimj